server/rest/license.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from girder.api import access
+from girder.api.describe import Description, describeRoute
+from girder.api.rest import Resource
+
+
+class License(Resource):
+
+    def __init__(self):
+        super(License, self).__init__()
+        self.resourceName = 'license'
+        self.route('GET', (), self.get_licenses)
+
+    @access.public
+    @describeRoute(
+        Description('Return the licenses that a Tale can be under.')
+        .notes('This endpoint returns a list of all of the Whole Tale supported licenses')
+    )
+    def get_licenses(self, params):
+        return [
+            {
```

Comment:
At some point, should we consider moving these into the database?  For now I think this is fine.

Reply:
I'd suggest a different approach for now:
1. Wrap it in a class (`WholeTaleLicenses`?). Don't worry about mutability and let that array be an attribute of it (I don't have a good name for it... `licenses`).
2. In this method you'd just return `WholTaleLicenses.licenses`
3. Add a method to the class that would return supported *spdx*es. I think it'll be useful in couple of places (tests, validation etc.) e.g.:
  ```
  def supported_spdxes(self):
      return {license['spdx'] for license in self.licenses}
  ```